src/test/software/amazon/event/ruler/CIDRTest.java

```diff
@@ -241,7 +242,7 @@ public void TestCorrectRangesForSingleIpAddress() {
             tryAddingAsRule(m, c);
             assertEquals(wanted[i], c.toString());
         }
-        assertTrue(!m.isEmpty());
+        assertFalse(m.isEmpty());
```

Comment:
This looks weird.

Reply:
its checking to make sure once rules are added, there are state transitions. didn't see anything wrong w it other than method name, but I'm avoiding renaming methods until I get all the automated changes done.